### PR TITLE
LF-Repl: run only scenarios marked as test

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by sofiafaro-da — APPROVED** [review]
Thanks!

File: daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala [hunk 1]
```diff
@@ -510,7 +510,7 @@ object Repl {
           case None =>
             println("Error: " + id + " not found.")
             None
-          case Some(DValue(_, _, body, _)) =>
+          case Some(DValue(_, _, body, true)) =>
```
Comment (sofiafaro-da) [inline daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala:513]: Hm, is this `true` supposed to be here? This `buildExpr` function doesn't strike me as being for test scenarios only?

Comment (cocreature) [reply daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala:513]: It’s only used for testing and profiling scenarios.

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala:513]: I renamed the function

Comment (sofiafaro-da) [reply daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala:513]: Thanks! 👍

**Pull request merged**